### PR TITLE
Fixes the LevitIntegrationTest

### PR DESCRIPTION
**Pull request by AnugunjNaman**
There was a mismatch of logits which wasn't corrected. It's done now. 
@NielsRogge

### CODE REVIEW

**Review by NielsRogge — APPROVED** [review]
LGTM, cc @ydshieh
